go/vt/wrangler/testlib/rpc_reparent_external_test.go

```diff
@@ -589,6 +592,50 @@ func TestRPCTabletExternallyReparentedRerun(t *testing.T) {
 
 }
 
+func TestRPCTabletExternallyReparentedDemotesMasterToConfiguredTabletType(t *testing.T) {
```

Comment:
This test becomes obsolete along with the RPC. The new test should go into https://github.com/vitessio/vitess/blob/master/go/vt/wrangler/testlib/external_reparent_test.go

Reply:
Done